src/main/java/org/algorithms/ContainsDuplicate.java

```diff
@@ -0,0 +1,26 @@
+package org.algorithms;
+
+import java.util.Arrays;
+
+public class ContainsDuplicate {
+    public boolean containsDuplicate(int[] nums) {
+        Arrays.sort(nums);
+        int temp = nums.length-1;
+
+        while(temp > 0) {
```

Comment:
Replace `while` with `for` not to use extra temp variable. Using an internal var inside `for` is ok.